user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Travis CI Enterprise Operations manual
+layout: en_enterprise
+
+---
+
+This document is meant as a guideline to follow in case your Travis CI Enterprise production installation becomes unresponsive or unavailable.
+
+If this happens, we'd like to ask you to follow the steps outlined in this document first before you get in touch with us to save important data and collect as much evidence as possible. This will help us to provide you the best support possible and focus completely on resolving the problem.
+
+## First: Don't panic
```

Comment:
PERSONALLY I'd find this first step patronizing, but ymmv

If we're talking about them as steps, they should be numbered, as well.

Reply:
The only ones that are numberd steps are 

1. Back up your key
2. Back up your data directories
3. Pull a support bundle (can we call this "create" not "pull")
4. Write a support request

Reply:
Hmm... actually, I think I agree with @plaindocs. What if we moved the sentence into "coordinating with support" - that way it becomes more of a statement about support working with you, etc. 

The thing is, I know what customer experiences have made us decide that a reassurance is necessary, _but_, I agree - better to build it in as encouragement rather than a "stop what you're doing" message